src/components/utils/test/thread_test.cc

```diff
@@ -40,6 +40,7 @@ namespace utils {
 
 using namespace sync_primitives;
 using namespace threads;
+using threads::Thread;
 
 // TODO(AByzhynar): Add multithreading tests
```

Comment:
@AGritsevich These test use spawn only one additional thread. I think there should be tests with several threads to remove the comment.